addons/armaos/CfgOsFunctions.hpp

```diff
@@ -175,4 +175,18 @@ class CfgOsFunctions
 		man = "$STR_AE3_ArmaOS_Config_CommandFindMan";
 		code = "_this call AE3_armaos_fnc_os_find";
 	};
+	class mount : OsFunction
+	{
+		path = "/bin/mount";
+		description = "$STR_AE3_ArmaOS_Config_CommandFindDescr";
```

Comment:
"description" and "man" are both taken from the `find` command. It's the same for `mount` and `unmount`.